### ISSUE TITLE
[Test] E2e Tests for Notebook Examples

### ISSUE THREAD

**Issue by Electronic-Waste** [kubeflow/katib#2417]
### What you would like to be added?

We plan to add e2e tests for notebooks in CI/CD, run with papermill.

REF: https://github.com/nteract/papermill

### Why is this needed?

This will help us ensure the correctness of our notebook examples for data scientists.

### Love this feature?

Give it a 👍 We prioritize the features with most 👍

**Comment by yehudit1987** [comment]
I am interested in solving this
/assign

**Comment by gjyotin305** [comment]
Hi I am interested in adding this feature, could you tell me which notebooks should I start with ? @Electronic-Waste @andreyvelich 

/assign

**Comment by Electronic-Waste** [comment]
Sorry @gjyotin305, this issue has already been taken by @yehudit1987. 

I'm glad to help you if you pick another issue:)

**Comment by gjyotin305** [comment]
> Sorry @gjyotin305, this issue has already been taken by @yehudit1987. 
> 
> 
> 
> I'm glad to help you if you pick another issue:)

Ok

**Comment by Electronic-Waste** [comment]
Hi @yehudit1987, are you still working on this?

**Comment by andreyvelich** [comment]
@Electronic-Waste As you can see @saileshd1402 worked on the E2Es for Notebooks as part of this PR: https://github.com/kubeflow/training-operator/pull/2274.
Can we consolidate effort to build consistent solution for Katib and Training Operator ?
Maybe we can start with Training Operator, and after it setup the same workflow for Katib.

**Comment by Electronic-Waste** [comment]
@andreyvelich I agree with you. I'll look into the CI for Training Operator first.

And also, this is a kind reminder that PR #2437 is blocked by this issue. Looking forward to your earliest contributions @yehudit1987. I'm happy to help you with any questions:)

**Comment by chahatsagarmain** [comment]
@Electronic-Waste I would like to work on this issue . 
/assign
